lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -88,7 +89,8 @@ function contentDescription(content) {
 export default function FilePickerApp({ onSubmit }) {
   const submitButton = /** @type {{ current: HTMLInputElement }} */ (useRef());
   const {
-    filePicker: { formAction, formFields, ltiLaunchUrl, blackboard, canvas },
+    deeplinkingAPI,
+    filePicker: { formAction, formFields, blackboard, canvas },
```

Comment:
According to the types, `deeplinkingAPI` is part of the `filePicker` property rather than the top-level object. Which is correct?

Reply:
I think it makes sense inside `filePicker`, move it there everywhere.